lib/load.php

```diff
@@ -130,6 +130,7 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/wordpress-6.1/class-wp-theme-json-6-1.php';
 require __DIR__ . '/compat/wordpress-6.1/block-template-utils.php';
 require __DIR__ . '/compat/wordpress-6.1/wp-theme-get-post-templates.php';
+require __DIR__ . '/compat/wordpress-6.1/script-loader.php';
```

Comment:
Looks like `script-loader.php` has already been loaded above.

Reply:
doh! fixed.

Reply:
I think this is good to merge now @ramonjd, but given the mistake with the rebase, might not hurt to give it another quick look over when you have a minute, please.